modules/apps/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp

```diff
@@ -830,7 +799,7 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
 							name='<%= "ExpandoAttribute--" + mapDisplayName +"--" %>'
 						/>
 
-						<aui:script require="map-common/js/MapBase.es as MapBase">
+						<aui:script require="@liferay/map-common/js/MapBase.es as MapBase">
```

Comment:
Is this snippet actually working? As you mentioned, you'd need to explicitly pass in the package version number.

Since you're already creating a component for it, why not add this listener directly in there as well?

Reply:
:thinking: I think it should work thanks to bundler-imports, [since `map-common` is declared in the global `npmscripts.config.js`](https://github.com/liferay/liferay-portal/blob/master/modules/npmscripts.config.js#L265).

Reply:
OK, didn't see @bryceosterhaus comment in the PR description. So, if this doesn't work, maybe it's time to fix it :sweat_smile:  I'll prioritize that :+1:

Reply:
I don't think there's anything to prioritize here. We don't bundlerize bits of JS inside a JSP.  The approach is to move this into the component same as we did with the other code.

Reply:
That will be -> https://issues.liferay.com/browse/LPS-140010